src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        src="/src/images/logo.png"
+        alt="Logo"
+        class="logo-item">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="index.html" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="index.html"
+            class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
same here

Reply:
after fixing the attribute view, the tests just started skipping two tests, didn't change anything
![Screenshot at Dec 22 10-25-45](https://github.com/mate-academy/layout_moyo-header/assets/144326045/be966ac0-1533-4cc1-9798-0283278b976d)

Reply:
fixed